src/nvim/buffer.c

```diff
@@ -5085,20 +5085,22 @@ linenr_T buf_change_sign_type(
 }
 
 int buf_getsigntype(
-        buf_T *buf,
-        linenr_T lnum,
-        int type /* SIGN_ICON, SIGN_TEXT, SIGN_ANY, SIGN_LINEHL */
-        )
+    buf_T *buf,
+    linenr_T lnum,
+    int type  // SIGN_ICON, SIGN_TEXT, SIGN_ANY, SIGN_LINEHL, SIGN_NUMHL
```

Comment:
when reformatting a signature we prefer to change it to a single line (if it fits), and add doxygen comments like `/// @param type ...` in front.

Reply:
Done, is this what you had in mind? And should I add doxygen comments to `sign_get_attr` as well?